src/Symfony/Component/Ldap/Tests/Adapter/ExtLdap/AdapterTest.php

```diff
@@ -39,12 +39,39 @@ public function testLdapEscape()
      */
     public function testSaslBind()
     {
+        $h = @ldap_connect(getenv('LDAP_HOST'), getenv('LDAP_PORT'));
+        @ldap_set_option($h, \LDAP_OPT_PROTOCOL_VERSION, 3);
+
+        if (!$h || !@ldap_bind($h)) {
+            $this->markTestSkipped('No server is listening on LDAP_HOST:LDAP_PORT');
+        }
+
+        if (!@ldap_start_tls($h)) {
+            ldap_unbind($h);
+            $this->markTestSkipped('Cannot establish an encrypted connection');
```

Comment:
Spent an hour investigating on this and I have the same conclusion. I tried a few things to enable TLS with bitnami/openldap but it would require additional work, more than just turning TLS switch to "on" it seems.